dockerfiles/nginx/proxito.conf

```diff
@@ -64,6 +64,8 @@ server {
         add_header X-RTD-Redirect $rtd_redirect always;
         set $cache_tag $upstream_http_cache_tag;
         add_header Cache-Tag $cache_tag always;
+        set $referrer_policy $upstream_http_referrer_policy;
+        add_header Referrer-Policy $referrer_policy always;
```

Comment:
This needs a corresponding `ops` change as well.

Reply:
Yes it does although the critical piece is getting the stricter policy for the dashboard.

Reply:
I added a PR for the corresponding ops change.